Common/CPUDetect.cpp

```diff
@@ -255,6 +255,8 @@ void CPUInfo::Detect() {
 			if ((cpu_id[1] >> 11) & 1)
 				bRTM = true;
 		}
+
+		bBMI2_fast = bBMI2 && (vendor != VENDOR_AMD || family >= 0x19);
```

Comment:
Hah, I was gonna comment that we want to disable BMI2 on AMD before Zen 3, but you beat me to it.